go-mode.el

```diff
@@ -1835,7 +1835,7 @@ If ARG is non-nil, anonymous functions are skipped."
 If ARG is non-nil, anonymous functions are skipped."
   (interactive "P")
   (go-goto-function-name arg)
-  (forward-word 1)
+  (forward-word-strictly 1)
```

Comment:
https://lists.gnu.org/archive/html/emacs-diffs/2016-02/msg00108.html

Its function was introduced by above commit at 2016/FEB. Only Emacs 25 RC or development Emacs users can use it now.